anime-list-master.xml

```diff
@@ -1073,12 +1073,19 @@
     <name>Dragon Ball</name>
     <mapping-list>
       <mapping anidbseason="0" tvdbseason="0">;1-5;2-6;3-0;</mapping>
+      <mapping anidbseason="1" tvdbseason="2" start="13" end="28" offset="-13"/>
```

Comment:
Is this offset correct? Seems it's one episode too high

Reply:
The offset was correct, but the start value was not. Fixed the issue.